### DISCUSSION TITLE
Wiki page to show key places to customize the code for branding

### DISCUSSION THREAD

**Discussion by Grant8907** [IBM/mac-ibm-notifications#13] [Ideas]
Feature request came in from Slack:

Would be good to add a Wiki page on how to rebrand the app for those that want to compile their own version

**Comment by SMartorelli** [comment]
We'll work to release it asap!

**Comment by SMartorelli** [comment]
Created an [issue](https://github.com/IBM/mac-ibm-notifications/issues/15) for this and added it to the [April milestone](https://github.com/IBM/mac-ibm-notifications/milestone/1).

**Reply by jvalentik** [reply]
Wiki published [here](https://github.com/IBM/mac-ibm-notifications/wiki/Rebranding-the-application)
